Automattic/wp-calypso
# Launch Site: Add email confirmation step

#### Proposed Changes

* As we've discussed( p58i-bwV-p2 ), users will be asked to verify their email address to launch their site.

#### Testing Instructions



* Create a new user.
* Keep the email unverified.
* Create a new website. In My Home setup list, the `Launch site` button should be disabled.
  * The button should be enabled when the email is verified.
   <img width="1056" alt="image" src="https://user-images.githubusercontent.com/212034/177249595-fd57495d-735b-42f0-b0f1-bec11136c616.png">
* Try to launch the website via `Visit site` > `Launch Site`.
  <img width="564" alt="image" src="https://user-images.githubusercontent.com/212034/177249772-5d1d6631-5cc7-41cb-b941-e22756531fd3.png">
  <img width="515" alt="image" src="https://user-images.githubusercontent.com/212034/177249822-1df5cded-4d88-4295-9f7f-cb63897a96af.png">
* You will be see the email confirmation step.
  * This screen should not show up if your email has been verified.
  <img width="1260" alt="image" src="https://user-images.githubusercontent.com/212034/177250026-abe30415-fb26-48f9-81fc-ce7b584f8113.png">
